Dashboard/src/routes-nav/Navigation.js

```diff
@@ -1,51 +1,94 @@
-import React, { useState } from "react"
+import React from "react"
+import { useHistory } from "react-router-dom"
 import {
-  Collapse,
-  Navbar,
-  NavbarToggler,
-  NavbarBrand,
-  Nav,
-  NavItem,
-  NavLink,
-} from "reactstrap"
+  BottomNavigation,
+  BottomNavigationAction,
+  makeStyles,
+} from "@material-ui/core"
+import Computer from "@material-ui/icons/Computer"
+import FeaturedPlayListIcon from "@material-ui/icons/FeaturedPlayList"
+import AccountCircle from "@material-ui/icons/AccountCircle"
+import CreditCard from "@material-ui/icons/CreditCard"
+import AccountBalance from "@material-ui/icons/AccountBalance"
+import helixLogo from "../helixLogo.jpg"
 
-/** @returns {jsx} Handles Navigation for Helix Regulatory DashBoard. */
-const Navigation = () => {
-  // When the browser window gets to small, nav bar collapses and toggle is used to show and hide the navbar.
-  const [isOpen, setIsOpen] = useState(false)
-  const toggle = () => setIsOpen(!isOpen)
+const useStyles = makeStyles({
+  root: {
+    display: "grid",
+  },
 
-  return (
-    <div>
-      <Navbar color="light" light expand="md">
-        <NavbarBrand href="/">
-          <b>Home</b>
-        </NavbarBrand>
-        <NavbarToggler onClick={toggle} />
-        <Collapse isOpen={isOpen} navbar>
-          <Nav className="ml-auto" navbar>
-            <NavItem>
-              <NavLink href="/dashboard">Dashboard</NavLink>
-            </NavItem>
+  homeButton: {
+    backgroundColor: "white",
+    border: "none",
+    width: "300px",
+    marginTop: "15px",
+    gridArea: "1 / 1 / row1-end / 3",
+  },
 
-            <NavItem>
-              <NavLink href="/entity">Entity</NavLink>
-            </NavItem>
+  hello: {
+    marginTop: "15px",
+    marginRight: "15px",
+    gridArea: "1 / 4 / row1-end / 6",
+  },
+})
 
-            <NavItem>
-              <NavLink href="/loan">Loan</NavLink>
-            </NavItem>
+/** @returns {jsx} Handles Navigation for Helix Regulatory DashBoard. */
+const Navigation = () => {
+  const history = useHistory()
+  const classes = useStyles()
+  const [value, setValue] = React.useState(0)
 
-            <NavItem>
-              <NavLink href="/regulatory">Regulatory</NavLink>
-            </NavItem>
+  const handleHome = () => {
+    setValue(0)
+    history.push(`/`)
+  }
 
-            <NavItem>
-              <NavLink href="/myrequest">My Request</NavLink>
-            </NavItem>
-          </Nav>
-        </Collapse>
-      </Navbar>
+  return (
+    <div>
+      <div className={classes.root}>
+        <button
+          onClick={handleHome}
+          type="button"
+          className={classes.homeButton}
+        >
+          <img src={helixLogo} alt="helixLogo" />
+        </button>
+        <BottomNavigation
```

Comment:
Lets use bottom nav for a footer component (haven't really talked about that yet). Check out AppBar https://material-ui.com/components/app-bar/